collect_app/src/test/java/org/odk/collect/android/draw/PenColorPickerViewModelTest.kt

```diff
@@ -0,0 +1,42 @@
+package org.odk.collect.android.draw
+
+import android.graphics.Color
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.odk.collect.settings.keys.MetaKeys
+import org.odk.collect.shared.settings.InMemSettings
+import org.odk.collect.shared.settings.Settings
+
+@RunWith(AndroidJUnit4::class)
+class PenColorPickerViewModelTest {
+    private lateinit var metaSettings: Settings
+    private lateinit var viewModel: PenColorPickerViewModel
+
+    @Before
+    fun setup() {
+        metaSettings = InMemSettings()
+        viewModel = PenColorPickerViewModel(metaSettings)
+    }
+
+    @Test
+    fun `default pen color should be black`() {
+        assertThat(viewModel.penColor.value, `is`(Color.BLACK))
+    }
+
+    @Test
+    fun `last used pen color should be saved in meta settings`() {
```

Comment:
This should really be `setPenColor sets penColor`

Reply:
Done.